src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -30,6 +31,51 @@ class Emails_Tab {
 	 * @var string
 	 */
 	public static $slug = 'emails';
+	
+	/**
+	 * Holder for template object.
+	 *
+	 * @since TBD
+	 *
+	 * @var null|Tribe_Template
+	 */
+	protected $template;
+
+	/**
+	 * Edit section slug.
+	 * 
+	 * @since TBD
+	 * 
+	 * @var string
+	 */
+	public static $edit_section_slug = 'edit_email';
+
+	/**
+	 * Editing id key.
+	 * 
+	 * @since TBD
+	 * 
+	 * @var string
+	 */
+	public static $key_edit_id = 'id';
+
+	/**
+	 * Key to determine current section.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $key_current_section = 'tec_tickets_emails_current_section';
```

Comment:
The current section should come from the get variable `section`

Reply:
This is a variable for a hidden input field that is posted when settings are saved. The `section` variable is not available when the settings are posted. I ran into this issue when creating the provider sections for Tickets Commerce.